src/subsystems/haskell/translators/stack-lock/default.nix

```diff
@@ -114,14 +114,14 @@ in {
         (l.attrNames projectTree.files);
 
       cabalFile = projectTree.getNodeFromPath (l.head cabalFiles);
-      cabal = haskellUtils.fromCabal cabalFile.fullPath project.name;
+      cabal = stackLockUtils.fromCabal cabalFile.fullPath project.name;
```

Comment:
See point 1 in https://github.com/nix-community/dream2nix/pull/228#issue-1330391847.